app/_src/guides/excluding-traffic-from-transparent-proxy-redirection.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Excluding Traffic from Transparent Proxy Redirection
+content_type: tutorial
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+
+This guide will show you how to exclude specific types of traffic from being redirected by the transparent proxy. It includes instructions for both **Universal** and a **Kubernetes** modes and covers different types of traffic you might want to exclude, such as:
+
+- [Exclude traffic on certain ports (incoming & outgoing)](#exclude-traffic-on-certain-ports)
+- [Exclude traffic to and from specific IP addresses](#exclude-traffic-to-and-from-specific-ip-addresses)
+
+Use this guide to control which traffic the transparent proxy intercepts and which it allows through directly.
+
+## Terminology overview
```

Comment:
should we split this terminology overview between kubernetes and universal? Do we even need it, I guess these are core concepts of Kubernetes and users should be familiar with them when they start playing with service mesh